examples/tox21/test_tox21.sh

```diff
@@ -2,19 +2,14 @@
 
 set -e
 
-gpu=-1
+# gpu id given from first argument, default value is -1
+gpu=${1:--1}
 
-declare method_list=(nfp ggnn schnet weavenet rsgcn)
 # Preprocessor parse result must contain both pos/neg samples
 tox21_num_data=100
 
-#for method in nfp ggnn schnet weavenet rsgcn
-for (( i=0; i<${#method_list[@]}; i++ ));
+for method in nfp ggnn schnet weavenet rsgcn
```

Comment:
Do you have any specific reason why you do not use the combination of `declare` and `for` (both are OK, I'm just curious).

Reply:
When I need to use `i`, I needed to `declare`.
But now `i` is not used and I thought it is more easy to understand bash script using only `for ... in ...` notation.

Reply:
OK understood.